pygeos/test/test_predicates.py

```diff
@@ -90,3 +90,11 @@ def test_equals_exact_tolerance():
     # default value for tolerance
     assert pygeos.equals_exact(p1, p1).item() is True
     assert pygeos.equals_exact(p1, p2).item() is False
+
+
+def test_relate():
```

Comment:
Can you also test with `None` input?

Reply:
Ah, good you ask, because I missed handling the second input for None